db/controllers.js

```diff
@@ -15,3 +19,12 @@ exports.getReviews = (request, response, next) => {
     })
     .catch(next);
 };
+
+exports.getReviewById = (request, response, next) => {
+  const { review_id } = request.params;
+  fetchReviewById(review_id)
+    .then((review) => {
+      response.status(200).send(review);
```

Comment:
Good stuff, one small thing is that you should be returning an object with the key of review and value of the review you've fetched - (this will unfortunately mean you'll need to alter your tests slightly to reflect this)